testsuite/domain/pom.xml

```diff
@@ -305,7 +302,7 @@
             <id>ee9.profile</id>
             <activation>
                 <property>
-                    <name>ts.ee9</name>
+                    <name>DISABLED.ts.ee9</name>
```

Comment:
I'm also going to revert this since based on our CI configuration, by using ee9 we will be testing wildfly preview with JDK 17, so it makes sense to me to leave it by now.